tbtc-ts/src/bitcoin.ts

```diff
@@ -110,13 +184,99 @@ export interface Client {
    */
   getRawTransaction(transactionHash: string): Promise<RawTransaction>
 
+  /**
+   * Gets the number of confirmations that a given transaction has accumulated
+   * so far.
+   * @param transactionHash - Hash of the transaction.
+   * @returns The number of confirmations.
+   */
+  getTransactionConfirmations(transactionHash: string): Promise<number>
+
+  /**
+   * Gets height of the latest mined block.
+   * @return Height of the last mined block.
+   */
+  latestBlockHeight(): Promise<number>
+
+  /**
+   * Gets concatenated chunk of block headers built on a starting block.
+   * @param blockHeight - Starting block height.
+   * @param confirmations -  Number of confirmations (subsequent blocks) built
+   *                         on the starting block.
+   * @return Concatenation of block headers in a hexadecimal format.
+   */
+  getHeadersChain(blockHeight: number, confirmations: number): Promise<string>
+
+  /**
+   * Get proof of transaction inclusion in the block.
+   * @param txHash - Hash of a transaction.
+   * @param blockHeight - Height of the block where transaction was confirmed.
+   * @return Transaction inclusion proof in hexadecimal form.
+   */
```

Comment:
Is this correct to talk about "proof" here? I think this is just about a Merkle branch allowing to perform inclusion proof for given transaction.

Reply:
Done.